cjdns-bytes/src/utils.rs

```diff
@@ -1,150 +1,224 @@
 //! Parsing and serialization helpers.
```

Comment:
How about to put `#![deny(missing_docs)]` and add missing docs? It would be really great if we can document as much as possible.

Reply:
Didn't even thought about documenting utils, however they are `pub`. Got it

Reply:
Think this is your #4 here :)